### PR TITLE
fix: remove noise & warning when creating Atomic project

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/CDX-726

When creating a new Atomic project, it's pretty clean now!

### CODE REVIEW

File: packages/create-atomic/package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
   },
   "dependencies": {
     "minimist": "1.2.5",
-    "plop": "^3.0.1",
+    "plop": "^3.0.3",
```
Comment (ThibodeauJF) [inline packages/create-atomic/package.json:37]: https://github.com/plopjs/plop/commit/24ad3807928d5465df999e9ee2c23c56d11791b7 😎

File: packages/create-atomic/src/plopfile.ts [hunk 2]
```diff
@@ -77,21 +77,21 @@ export default function (plop: NodePlopAPI) {
...
       function installPackagesPrompt() {
-        return 'Installing packages 🚀';
+        return '🚀  Installing packages...';
       },
       function installPackages(answers) {
         return new Promise((resolve, reject) => {
           const {project} = answers as PromptsAnswers;
           const process = spawn(getPackageManager(), ['install'], {
-            stdio: 'inherit',
+            stdio: 'ignore',
```
Comment (ThibodeauJF) [inline packages/create-atomic/src/plopfile.ts:86]: I still see a loading indicator, but no more bash vomit

File: packages/create-atomic/package.json (hunk 1 above)
Comment (louis-bompart) [reply packages/create-atomic/package.json:37]: Nice!

File: packages/create-atomic/src/plopfile.ts (hunk 2 above)
Comment (louis-bompart) [reply packages/create-atomic/src/plopfile.ts:86]: Ok cool! If there's a loading indicator, we're good :)

File: packages/create-atomic/src/plopfile.ts [hunk 3]
```diff
@@ -77,21 +77,21 @@ export default function (plop: NodePlopAPI) {
         ],
       },
       function installPackagesPrompt() {
-        return 'Installing packages 🚀';
+        return '🚀  Installing packages...';
```
Comment (louis-bompart) [inline packages/create-atomic/src/plopfile.ts:80]: You should check the terminal supports an emoji before using it.
Check https://github.com/koala-interactive/is-emoji-supported (no transitive deps attached so it's pretty light)

**Review by louis-bompart — COMMENTED** [review]
Just check if the user's terminal support emojis before using one

File: packages/create-atomic/src/plopfile.ts (hunk 3 above)
Comment (ThibodeauJF) [reply packages/create-atomic/src/plopfile.ts:80]: I mean it could just remove the emojis if they can cause an issue. The gain is pretty low

Comment (louis-bompart) [reply packages/create-atomic/src/plopfile.ts:80]: > I mean we could just remove the emojis if they can cause an issue. The gain is pretty low

As you prefer really, the choice is yours.

**Review by olamothe — APPROVED** [review]

**Review by louis-bompart — APPROVED** [review]

**Pull request merged**